src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -292,8 +293,8 @@ def cleanRSE(self, rse):
                             msg = "RSE: %s Failed to purge nonEmpty directory: %s"
                             self.logger.error(msg, rse['name'], dirPfn)
                     except Exception as ex:
-                        msg = "Error while cleaning RSE: %s"
-                        msg += "Will retry in the next cycle. Err: %s"
+                        msg = "Error while cleaning RSE: %s."
+                        msg += " Will retry in the next cycle. Err: %s"
                         self.logger.debug(msg, rse['name'], str(ex))
```

Comment:
How about this general exception being reported as "debug" level? Shouldn't it be either error or warning (or maybe even exception?)?

Reply:
Any logging of `exception` level regarding something expected should be avoided. This is exactly what triggers all those traceback dumps and is exactly the main topic of this fix. So I would say, no, it is good as it is.

Reply:
Actually thinking twice  - it would be best if it was a warning